test/integration-test/transaction/src/test/java/org/apache/shardingsphere/integration/transaction/cases/cursor/OpenGaussCursorTestCase.java

```diff
@@ -0,0 +1,158 @@
+        executeWithLog(conn, "close test;");
+        executeWithLog(conn, cursorSQLCommand.getBroadcastTablesCursor());
+        fetch(conn, 10101);
+        fetch(conn, 10102);
+        fetch(conn, 10201);
+        fetch(conn, 10202);
+        fetchOver(conn);
+        fetchOver(conn);
+        executeWithLog(conn, "rollback;");
+    }
+    
+    private void broadcastTableCursorTest2(final Connection conn) throws SQLException {
+        executeWithLog(conn, "start transaction;");
+        executeWithLog(conn, cursorSQLCommand.getBroadcastTablesCursor2());
+        executeWithLog(conn, "close test;");
+        executeWithLog(conn, cursorSQLCommand.getBroadcastTablesCursor2());
+        fetch(conn, 10101);
+        fetch(conn, 10102);
+        fetch(conn, 10201);
+        fetch(conn, 10202);
+        fetchOver(conn);
+        fetchOver(conn);
+        executeWithLog(conn, "rollback;");
+    }
+    
+    private void broadcastAndSingleTablesCursorTest(final Connection conn) throws SQLException {
+        executeWithLog(conn, "start transaction;");
+        executeWithLog(conn, cursorSQLCommand.getBroadcastAndSingleTablesCursor());
+        executeWithLog(conn, "close test;");
+        executeWithLog(conn, cursorSQLCommand.getBroadcastAndSingleTablesCursor());
+        fetch(conn, 1);
+        fetch(conn, 2);
+        fetch(conn, 3);
+        fetch(conn, 4);
+        fetchOver(conn);
+        fetchOver(conn);
+        executeWithLog(conn, "rollback;");
+    }
+    
+    private void broadcastAndSingleTablesCursorTest2(final Connection conn) throws SQLException {
+        executeWithLog(conn, "start transaction;");
+        executeWithLog(conn, cursorSQLCommand.getBroadcastTablesCursor2());
+        executeWithLog(conn, "close test;");
+        executeWithLog(conn, cursorSQLCommand.getBroadcastTablesCursor2());
+        fetch(conn, 10101);
+        fetch(conn, 10102);
+        fetch(conn, 10201);
+        fetch(conn, 10202);
+        fetchOver(conn);
+        fetchOver(conn);
+        executeWithLog(conn, "rollback;");
+    }
+    
+    private void viewCursorTest(final Connection conn) throws SQLException {
+        executeWithLog(conn, "start transaction;");
+        executeWithLog(conn, cursorSQLCommand.getViewCursor());
+        executeWithLog(conn, "close test;");
+        executeWithLog(conn, cursorSQLCommand.getViewCursor());
+        fetch(conn, 1);
+        fetch(conn, 2);
+        fetch(conn, 3);
+        fetch(conn, 4);
+        fetchOver(conn);
+        fetchOver(conn);
+        executeWithLog(conn, "rollback;");
+    }
+    
+    private void fetch(final Connection conn, final int expectedId) throws SQLException {
+        ResultSet rs = executeQueryWithLog(conn, "fetch test;");
+        while (rs.next()) {
+            int id = rs.getInt("id");
+            Assert.assertEquals(String.format("Expected fetch id is %s, actual is %s.", expectedId, id), expectedId, id);
```

Comment:
Please use assertThat instead of assertEquals.

Reply:
OK,I will correct it.